proto/proto-backplane-grpc/src/main/proto/deephaven_core/proto/console.proto

```diff
@@ -100,11 +100,17 @@ message LogSubscriptionData {
     reserved 4;//if we can scope logs to a script session
     //    Ticket console_id = 4;
 }
+
 message ExecuteCommandRequest {
     io.deephaven.proto.backplane.grpc.Ticket console_id = 1;
     reserved 2;//if script sessions get a ticket, we will use this reserved tag
     string code = 3;
+
+    // If set to `true` the command will be executed systemically.  Failures in systemic code
+    // are treated as important failures and cause the server to shut down
+    optional bool systemic=4;
```

Comment:
I don't think we care that much, but just nit that all the others have space around `=`.

I _think_ that `optional` is a good qualifier here; essentially, this allows us to differentiate between "unset" vs "default false" (important in the case where we want might want the default to be `true`).

I think we should add a comment "if unset, is treated as `false`".